www/js/shortcuts/shortcuts-map.json

```diff
@@ -0,0 +1,15 @@
+{
+  "waheguru": ["mod+1"],
+  "moolMantra": ["mod+2"],
+  "empty": ["mod+3"],
+  "anandSahibBhog": ["mod+4"],
+  "help": ["mod+5"],
+  "legend": ["mod+6"],
+  "searchBar": ["mod+/"],
+  "showDhanGuruModal": ["mod+g"],
+  "prevLine": ["up", "left"],
+  "nextLine": ["down", "right"],
+  "spaceBar": ["space"],
```

Comment:
the name for this action could be more meaningful. guess updating it would require updating the function as well.